### PR TITLE
Add Ryn's Whiterun City Limits

### PR DESCRIPTION
**Pull request by loicreynier**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -21763,6 +21763,34 @@ plugins:
...
         util: 'SSEEdit v4.0.4'
 
+  - name: 'RynsWhiterunCityLimits.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65661/'
+        name: 'Ryn''s Whiterun City Limits'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'eFPS'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
          - 'eFPS - Exterior FPS boost'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -21763,6 +21763,34 @@ plugins:
...
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Bleakwind Basin.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Bleakwind_Basin_Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Ryn''s Halted Stream Camp'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Halted Stream Camp.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Halted_Stream_Camp_PATCH.esp")'
+      - <<: *patchProvided
+        subs:
+          - 'Tundra Homestead'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
          - 'CC Tundra Homestead'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -21763,6 +21763,34 @@ plugins:
...
+  - name: 'RynsWhiterunCityLimits.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65661/'
+        name: 'Ryn''s Whiterun City Limits'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'eFPS - Exterior FPS boost'
+          - '[Unofficial eFPS patches](https://www.nexusmods.com/skyrimspecialedition/mods/70176/)'
+        condition: 'active("Occ_Skyrim_Tamriel.esp") and not active("Occ_Skyrim_RynsWhiterunCityLimits_patch")'
```
Comment (MacSplody) [inline masterlist.yaml]: Missing file extension.
```suggestion
        condition: 'active("Occ_Skyrim_Tamriel.esp") and not active("Occ_Skyrim_RynsWhiterunCityLimits_patch.esp")'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -21763,6 +21763,34 @@ plugins:
...
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Bleakwind Basin.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Bleakwind_Basin_Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Ryn''s Halted Stream Camp'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Halted Stream Camp.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Halted_Stream_Camp_PATCH.esp")'
+      - <<: *patchProvided
+        subs:
+          - 'CC Tundra Homestead'
```
Comment (sibir-ine) [inline masterlist.yaml]: Single-element lists can be written in flow style:
```suggestion
        subs: [ 'CC Tundra Homestead' ]
```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**